assets/static/ui.js

```diff
@@ -4,49 +4,49 @@ var UI = (function(params) {
     // when user click on any alert label modal popup with a list of possible
     // filter will show, this function is used to setup that modal
     setupModal = function() {
-        $('#labelModal').on('show.bs.modal', function(event) {
+        $("#labelModal").on("show.bs.modal", function(event) {
             Unsee.Pause();
             var modal = $(this);
             var label = $(event.relatedTarget);
-            var label_key = label.data('label-key');
-            var label_val = label.data('label-val');
+            var label_key = label.data("label-key");
+            var label_val = label.data("label-val");
             var attrs = Alerts.GetLabelAttrs(label_key, label_val);
             var counter = Summary.Get(label_key, label_val);
-            modal.find('.modal-title').html(
-                Templates.Render('modalTitle', {
+            modal.find(".modal-title").html(
+                Templates.Render("modalTitle", {
                   attrs: attrs,
                   counter: counter
                 })
             );
             var hints = Autocomplete.GenerateHints(label_key, label_val);
-            modal.find('.modal-body').html(
-              Templates.Render('modalBody', {hints: hints})
+            modal.find(".modal-body").html(
+              Templates.Render("modalBody", {hints: hints})
             );
-            modal.on('click', '.modal-button-filter', function(elem) {
-                var filter = $(elem.target).data('filter-append-value');
-                $('#labelModal').modal('hide');
+            modal.on("click", ".modal-button-filter", function(elem) {
+                var filter = $(elem.target).data("filter-append-value");
+                $("#labelModal").modal("hide");
                 Filters.AddFilter(filter);
             });
         });
-        $('#labelModal').on('hidden.bs.modal', function(event) {
+        $("#labelModal").on("hidden.bs.modal", function(event) {
             var modal = $(this);
-            modal.find('.modal-title').children().remove();
-            modal.find('.modal-body').children().remove();
+            modal.find(".modal-title").children().remove();
+            modal.find(".modal-body").children().remove();
             Unsee.WaitForNextReload();
         });
     }
 
 
     // each alert group have a link generated for it, but we hide it until
-    // user hovers over that group so it doesn't trash the UI
+    // user hovers over that group so it doesn"t trash the UI
```

Comment:
I was fixing quotation marks and got one too many, it's a comment so I won't let that bother me much